modules/persistent-storage-csi-vsphere-increase-max-vols-per-node-procedure.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-increase-max-vols-per-node_{context}"]
+= Increasing the maximum allowable volumes per node for vSphere
+
+.Prerequisites
+* Access to the {product-title} web console.
+
+* Access to the cluster as a user with the cluster-admin role.
+
+* Access to VMware vSphere vCenter.
+
+* In vCenter, ensure that the parameter `pvscsiCtrlr256DiskSupportEnabled` is set to 'True'. For information about how to set this parameter, see VMware vSphere vCenter documentation under the _Additional Resource_ section.
```

Comment:
This is not enough, we need to explicitly mention that this parameter is not fully supported by VMware. The link in the additional resources does not help with that.